src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.VirtualFree.cs

```diff
@@ -10,6 +10,6 @@ internal partial class Interop
     internal partial class Kernel32
     {
         [DllImport(Libraries.Kernel32)]
-        internal static extern unsafe void* VirtualAlloc(void* address, UIntPtr numBytes, int commitOrReserve, int pageProtectionMode);
+        unsafe internal static extern bool VirtualFree(void* address, UIntPtr numBytes, int pageFreeMode);
```

Comment:
Nit: We generally match the argument names from the official headers. So the argument names here would be `lpAddress, dwSize, dwFreeType`.

Similar for the other PInvokes.

Reply:
It was actually taken verbatim from the CoreRT code, but since I need to modify the shared code anyway I will go and fix it.